docs/product-installation-guide/src/main/asciidoc/eap-dm-install-proc.adoc

```diff
@@ -1,20 +1,41 @@
 [id='eap-dm-install-proc']
-= Installing {CENTRAL} using the deployable zip file
+= {CENTRAL} ZIP file installation
+
 {CENTRAL} is a web console that enables you to perform the following tasks over individual components in a unified web-based environment:
 
 * Create, manage, and edit your rules and related assets.
-* Manage connected Decision Servers and their containers.
+* Manage connected {KIE_SERVER} instances and their containers.
```

Comment:
There should be also third point, Business central allows executing runtime operations against processes and tasks in Kie servers connected to the Business Central.